starter-code/src/components/Tab.jsx

```diff
@@ -0,0 +1,21 @@
+import React, {Component} from 'react';
+import Buttons from './Buttons';
+
+class Tab extends Component {
+
+  render() {
+    const val = this.props.inf;
+
+    return (
+    <tr>
+      <th><img className="imagem" src={val.pictureUrl} /></th>
+      <th>{val.name}</th>
+      <th>{val.popularity}</th>
+      <Buttons functionClick={this.deleteItem} text={val.name}>Delete</Buttons>
```

Comment:
aqui você usa 'this.deleteItem', isso não existe neste componente, sua função foi passada como props com o nome anterior então está dentro de `this.props.fuctionClick`